plugins/flatpak/gs-flatpak-transaction.c

```diff
@@ -62,6 +62,52 @@ gs_flatpak_transaction_set_no_deploy (FlatpakTransaction *transaction, gboolean
 	g_object_notify (G_OBJECT (self), "no-deploy");
 }
 
+/* Sets installed app(s) back to installed state. Since libflatpak can return apps as updatable
```

Comment:
Could say "back to installed state when the associated runtime has been installed"